server/src/main/java/org/opensearch/index/IndexService.java

```diff
@@ -252,7 +253,8 @@ public IndexService(
         CompositeIndexSettings compositeIndexSettings,
         Consumer<IndexShard> replicator,
         Function<ShardId, ReplicationStats> segmentReplicationStatsProvider,
-        Supplier<Integer> clusterDefaultMaxMergeAtOnceSupplier
+        Supplier<Integer> clusterDefaultMaxMergeAtOnceSupplier,
```

Comment:
Should we also include clusterDefaultMaxMergeAtOnce value inside the ClusterMergeSchedulerConfig?